packages/venia-ui/lib/components/AddressBookPage/addressBookPage.js

```diff
@@ -38,15 +54,11 @@ const AddressBookPage = props => {
                     key={addressEntry.id}
                     address={addressEntry}
                     countryName={countryName}
+                    onEdit={handleEditAddress}
                 />
             );
         });
-
-        // sort the collection so the default is first
-        return addresses.sort(address =>
-            address.props.address.default_shipping ? -1 : 1
-        );
```

Comment:
This code would sort the default address to the front but it made no guarantees about the order of the rest of the list.

While that's ultimately what the storefront wants, it caused issues during snapshot testing (the order of the rest of the list would sometimes change).